src/site/content/en/react/react.11tydata.js

```diff
@@ -7,46 +7,27 @@ module.exports = {
     slug: 'react',
     cover: '/images/collections/react.svg',
     title: 'React',
-    updated: 'April 28, 2019',
+    updated: 'October 11, 2019',
     description: `Build performant and progressive React applications.`,
     overview: `In this section you'll learn how to use built-in APIs and
     third-party libraries to improve the performance of your
     React applications.`,
     topics: [
       {
-        title: 'Introduction',
+        title: 'Next.js',
```

Comment:
I think since we're redoing this page it's ok, but did want to call out that changing these titles does break the hash anchors. Want to make sure folks are cool with that before we land.

Reply:
This is another example where auto-generated hash anchors are brittle. See #1676.

Reply:
Well yes and no. I think originally folks may have manually named that hash anchor `introduction` anyway. And it might be weird to have `#introduction` point at at Next.js thing. Though, maybe folks would have used #introduction to point at the CRA section? ¯\_(ツ)_/¯